src/main/java/com/example/digitalmenuapi/controller/AdminMenuController.java

```diff
@@ -0,0 +1,71 @@
+package com.example.digitalmenuapi.controller;
+
+import com.example.digitalmenuapi.model.AdminMenuItems;
+import com.example.digitalmenuapi.service.AdminMenuItemsService;
+import java.net.URI;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * admin controller class.
+ */
+@RestController
+@RequestMapping("/admin")
+public class AdminMenuController {
+
+  @Autowired
+  AdminMenuItemsService adminMenuItemsService;
+
+  public AdminMenuController(AdminMenuItemsService adminMenuItemsService)  {
+    this.adminMenuItemsService = adminMenuItemsService;
+  }
+
+  Logger logger = LoggerFactory.getLogger(MenuItemsController.class);
+
+  /**
+   * Post request method to add new sandwiches to the menu.
+   */
+  @PostMapping("/menu")
+  @PreAuthorize("hasRole('ADMIN')")
+  public ResponseEntity<AdminMenuItems> addMenu(@RequestBody AdminMenuItems adminMenuItems) {
+    AdminMenuItems addedMenu = adminMenuItemsService.createNewSandwiches(adminMenuItems);
+    logger.info("Sandwich created successfully");
+    return ResponseEntity.created(URI.create("admin/menu/" + addedMenu.getId())).body(addedMenu);
+  }
+
+  /**
+   * Delete request method to delete sandwiches from the menu
+   * providing the correct id.
+   */
+  @DeleteMapping()
+  @PreAuthorize("hasRole('ADMIN')")
+  public ResponseEntity<String> deleteMenu(@RequestParam String id) {
+    adminMenuItemsService.deleteMenuItem(id);
+    logger.info("Sandwich deleted successfully");
```

Comment:
What if there is no sandwich in the menu? would this error and then need to be caught?